src/components/MenuItem/MenuItem.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export default function MenuItem({ item }) {
+  // console.log(item)
+  const { id, name, price, shortDescription } = item;
+
+  return (
+    <div>
+     id: {id}<br/>name: {name},<br/>price: ${price.toFixed(2)},<br/>short Description: {shortDescription}
```

Comment:
Using `< br />` like this is a way to do it. I think it's more standard to use `<ul>` and `<li>`, then style the list and list items to look the way you want.